src/docs/timestamp-global.dox

```diff
@@ -54,10 +54,10 @@ To force data to disk, advance the stable timestamp as desired and
 take a checkpoint.
 
 \warning The \c durable_timestamp is not changed when the application calls the
-WT_CONNECTION::rollback_to_stable method, and likely should be reset by the
-application in that case.
+WT_CONNECTION::rollback_to_stable method, and should be reset by the application
+in that case.
 
-@subsection timestamp_global_set_api_oldest_timestamp Setting the "oldest_timestamp" timestamp
+@subsection timestamp_global_set_api_oldest_timestamp Setting the \c "oldest_timestamp" timestamp
 
```

Comment:
This should probably have either quotes or \c but not both. In the last batch of changes I was shooting for \c with the literal full name (with _timestamp) and nothing with just the descriptive name. That is, 'you can set \c oldest_timestamp with WT_SESSION::set_timestamp' and 'you can set the oldest timestamp with WT_SESSION::set_timestamp', respectively.

Obviously I missed a bunch :-|

Reply:
I think that was my mistake, not yours!

Fixed in 7c4e9a17e.